sormas-app/app/src/main/java/de/symeda/sormas/app/event/edit/EventValidator.java

```diff
@@ -0,0 +1,14 @@
+package de.symeda.sormas.app.event.edit;
+
+import de.symeda.sormas.app.backend.event.Event;
+import de.symeda.sormas.app.component.validation.ValidationHelper;
+import de.symeda.sormas.app.databinding.FragmentEventEditLayoutBinding;
+
+class EventValidator {
+
+	public static void initializeEventStartDateValidation(final Event event, final FragmentEventEditLayoutBinding contentBinding) {
+		if (event != null) {
+			ValidationHelper.initDateIntervalValidator(contentBinding.eventStartDate, contentBinding.eventEndDate);
+		}
+	}
+}
```

Comment:
Why did you create this new class, instead of just doing the nullcheck & adding the validator inline in `EventFragment.java`?

Reply:
Inlined.